aten/src/ATen/native/cuda/Math.cuh

```diff
@@ -466,6 +466,408 @@ const auto entr_string = jiterator_stringify(
+
+          return p * x * (z - Z1) * (z - Z2);
+      }
+
+      T w = T{5.0} / x;
+
+      T p = polevl(w * w, PP, 6) / polevl(w * w, PQ, 6);
+      T q = polevl(w * w, QP, 7) / polevl(w * w, QQ, 6);
+
+      p = p * cos(x - THREE_FRAC_PI_4) - w * q * sin(x - THREE_FRAC_PI_4);
+
+      return p * SQRT_FRAC_2_PI / sqrt(x);
+  }
+); // j1_string
+
+const auto y0_string = j0_string + jiterator_stringify(
+    T y0(T x) {
+        static const T PP[7] = {
+                +7.96936729297347051624e-04,
+                +8.28352392107440799803e-02,
+                +1.23953371646414299388e+00,
+                +5.44725003058768775090e+00,
+                +8.74716500199817011941e+00,
+                +5.30324038235394892183e+00,
+                +9.99999999999999997821e-01,
+        };
+
+        static const T PQ[7] = {
+                +9.24408810558863637013e-04,
+                +8.56288474354474431428e-02,
+                +1.25352743901058953537e+00,
+                +5.47097740330417105182e+00,
+                +8.76190883237069594232e+00,
+                +5.30605288235394617618e+00,
+                +1.00000000000000000218e+00,
+        };
+
+        static const T QP[8] = {
+                -1.13663838898469149931e-02,
+                -1.28252718670509318512e+00,
+                -1.95539544257735972385e+01,
+                -9.32060152123768231369e+01,
+                -1.77681167980488050595e+02,
+                -1.47077505154951170175e+02,
+                -5.14105326766599330220e+01,
+                -6.05014350600728481186e+00,
+        };
+
+        static const T QQ[7] = {
+                +6.43178256118178023184e+01,
+                +8.56430025976980587198e+02,
+                +3.88240183605401609683e+03,
+                +7.24046774195652478189e+03,
+                +5.93072701187316984827e+03,
+                +2.06209331660327847417e+03,
+                +2.42005740240291393179e+02,
+        };
+
+        static const T YP[8] = {
+                +1.55924367855235737965e+04,
+                -1.46639295903971606143e+07,
+                +5.43526477051876500413e+09,
+                -9.82136065717911466409e+11,
+                +8.75906394395366999549e+13,
+                -3.46628303384729719441e+15,
+                +4.42733268572569800351e+16,
+                -1.84950800436986690637e+16,
+        };
+
+        static const T YQ[7] = {
+                +1.04128353664259848412e+03,
+                +6.26107330137134956842e+05,
+                +2.68919633393814121987e+08,
+                +8.64002487103935000337e+10,
+                +2.02979612750105546709e+13,
+                +3.17157752842975028269e+15,
+                +2.50596256172653059228e+17,
+        };
+
+        static const T FRAC_2_PI = 0.636619772367581343075535053490057448;
+        static const T FRAC_PI_4 = 0.785398163397448309615660845819875721;
+        static const T SQRT_FRAC_2_PI = 0.797884560802865355879892119868763737;
```

Comment:
Nit. For constants, it would be better to have them as `constexpr T`, as forcing them to be static can sometimes be a pesimisation.